content/docs/introduction/central.md

```diff
@@ -0,0 +1,69 @@
++++
+title = "Open mSupply Central Server"
+description = "Open mSupply central server functionality"
+date = 2022-06-10T11:38:00+00:00
+updated = 2022-06-10T11:38:00+00:00
+draft = false
+weight = 8
+sort_by = "weight"
+template = "docs/page.html"
+
+[extra]
+toc = true
+top = true
++++
+
+Open mSupply central server is a special site that allows for configurations of subset of central data. Please see [requirements](/docs/introduction/requirements/#open-msupply-requirements) section for extra info.
+
+## What is open mSupply central server
+
+In essence it's just another open mSupply instance. What makes it difference from a typical remote site ?
+
+- Only one instance of open mSupply central server will exist in open mSupply system
+- It will be configured by our support staff and needs to be available on a world wide web (typically as a cloud server, but can also be hosted in country)
+- All remote instance of open mSupply will communicate with open mSupply central server, as part of the synchronisation process
+- Allows for configurations of subset of central data
+
+## How does open mSupply central server look
+
+It will look very much like any other open mSupply instance, you will see distinct coloured bar at the bottom of interface which identifies the site as central server
+
+![central server identifier bar!](/docs/introduction/images/central_server_bar_identifier.png)
+
+Similar to mSupply central server, only one instance of open mSupply central server will exist in open mSupply system. 
+Our support staff will be control 
+
+## Remote site vs central server
+
+Some operations are only allowed in open mSupply central server, if operations is prohibited on remote site you will see the following alert
+
+![only allowed on central!](/docs/introduction/images/only_allowed_on_central.png)
+
+Throughout this documentation you will see sections that refer to open mSupply central server functionality, they will look like this and will have a link to this page
+
+[![central server only](/docs/introduction/images/central_server.png '🔗 Available on central server only ')](/docs/introduction/central/#remote-site-vs-central-server)
```

Comment:
Probably don't need to mention this since the places that does mention OMS central server can just link here 🤔 

So far only CCEI has talked about OMS central server and it's just in text, so idk if the image is needed? Or have another sentence in `What does it look like` explaining the icon and text indicator

Reply:
Like in the pack variant configuration in item catalogue the pack variant editing is visible but only editable on central server, in the docs i thought it would be good to have this link, and consistently every time open mSupply central server is mentioned. Similar in assets when asset can be attached to a store on central, but on remote it will be auto attached to current store. So basically in general docs, we would have areas where things only exists or only possible on central server, and I think it's good to just have a way to flag that to the user.

> OMS central server and it's just in text

Yeah so the point of this link is to indicate it with some visual aid, and consistently across all mentions of central server

Maybe it's good to see it as an example, i'll add in the pack variant docs

How does that sound ?

Reply:
Sounds good to me (: